Casks/pdk.rb

```diff
@@ -1,15 +1,36 @@
 cask 'pdk' do
-  version '1.5.0.0'
+  version '1.6.0.0'
 
-  # TODO: change this to a case/switch once we add 10.13
-  if MacOS.version <= :el_capitan
-    sha256 '4de3c31b27aa0201a3c0eeff413a19349c10541854687ac3874aed49238a7983'
-    url "https://puppet-pdk.s3.amazonaws.com/pdk/#{version}/repos/apple/10.11/PC1/x86_64/pdk-#{version}-1.osx10.11.dmg"
+  version_map = {
+    high_sierra: { 
+      sha: '582b746fc07829f5e89c3ef5c937a86e318444f166ee0afa9b7add12d0acf2a6',
+      ver: '10.13'
+    },
+    sierra: {
+      sha: '304d7c765b82f3e552b2023c5401be85085abec2c99af4f16918cf7fd7007cbd',
+      ver: '10.12'
+    },
+    el_capitan: {
+      sha: 'b1c044e7425a28ea796c33876f6c8f43e5f1dc4c536d93a0b168cb14be1800ba',
+      ver: '10.11'
+    }
+  }
+
+  case MacOS.version
+  when :high_sierra
+    sha = version_map[:high_sierra][:sha]
+    os_ver = version_map[:high_sierra][:ver]
+  when :sierra
+    sha = version_map[:sierra][:sha]
+    os_ver = version_map[:sierra][:ver]
   else
-    sha256 '713668b5113cf2d9309cfd199be0d9a52357d1c878c2aa8ef62ebb1ace55649b'
-    url "https://puppet-pdk.s3.amazonaws.com/pdk/#{version}/repos/apple/10.12/PC1/x86_64/pdk-#{version}-1.osx10.12.dmg"
+    sha = version_map[:el_capitan][:sha]
+    os_ver = version_map[:el_capitan][:ver]
   end
 
+  sha256 sha
+  url "https://puppet-pdk.s3.amazonaws.com/pdk/#{version}/repos/apple/#{os_ver}/PC1/x86_64/pdk-#{version}-1.osx#{os_ver}.dmg"
```

Comment:
I'm not 100% sure if ruby scoping will see `os_ver` here? It might -- if you know for sure then don't sweat it :)

Reply:
It will.